plexupdate.sh

```diff
@@ -444,7 +435,13 @@ if [ "${CHECKUPDATE}" = "yes" -a "${AUTOUPDATE}" = "no" ]; then
 	rm "${FILE_REMOTE}" 2>/dev/null >/dev/null
 fi
 
-
+getPlexServerToken() {
+  [ -f /etc/default/plexmediaserver ] && . /etc/default/plexmediaserver
+  local pmsApplicationSupportDir="${PLEX_MEDIA_SERVER_APPLICATION_SUPPORT_DIR:-${HOME}/Library/Application Support}"
+  local prefFile="${pmsApplicationSupportDir}/Plex Media Server/Preferences.xml"
+  
+  sed -n 's/.*PlexOnlineToken="\([[:alnum:]]*\).*".*/\1/p' "$prefFile"
```

Comment:
Maybe check that the file exists before assuming? Avoids errors or messages in bad situations.